File: app/src/main/java/sk/teamsoft/observableadapterdemo/DemoApp.java
package sk.teamsoft.observableadapterdemo;

import android.app.Application;

import timber.log.Timber;

/**
 * @author Dusan Bartos
 *         Created on 19.05.2017.
 */

public class DemoApp extends Application {

    @Override public void onCreate() {
        super.onCreate();
        Timber.plant(new Timber.DebugTree());
    }
}

File: app/src/main/java/sk/teamsoft/observableadapterdemo/DemoApp.kt
package sk.teamsoft.observableadapterdemo

import android.app.Application
import timber.log.Timber
import timber.log.Timber.DebugTree

/**
 * @author Dusan Bartos
 */
class DemoApp : Application() {
    override fun onCreate() {
        super.onCreate()
        Timber.plant(DebugTree())
    }
}